src/DssSpell.t.base.sol

```diff
@@ -37,6 +38,10 @@ struct TeleportGUID {
     uint48 timestamp;
 }
 
+interface AuthLike {
+    function wards(address) external view returns (uint256);
+}
+
```

Comment:
this shouldn't be required as there is already `WardsAbstract` available in the imported `dss-interfaces`